### PR TITLE
Removed unwarranted hyperlinks to OpenSim::Set

### LINKED ISSUE #601: [doxygen] Remove unwarranted hyperlinks to Set class
[Here's an example](https://github.com/opensim-org/opensim-core/blob/master/OpenSim/Common/Object.h#L264).

### PR DESCRIPTION
**Pull request by tkuchida**
Searched for `" Set "` in header files and prefixed with `%` where appropriate to remove hyperlinks.

Fixes #601
[ci skip]

### CODE REVIEW

File: OpenSim/Actuators/TorqueActuator.h [hunk 1]
```diff
@@ -114,18 +114,20 @@ class OSIMACTUATORS_API TorqueActuator : public ScalarActuator {
...
-    /** Set the 'optimal_force' property. **/
+    /** %Set the 'optimal_force' property. **/
     void setOptimalForce(double optimalForce)
     {   set_optimal_force(optimalForce); }
     /** Get the current value of the 'optimal_force' property. **/
     double getOptimalForce() const override // Part of Actuator interface.
     {   return get_optimal_force(); }
 
-    /* Set the bodies to which this actuator applies torque. */
+    /** %Set the first body to which this actuator applies torque. */
```
Comment (chrisdembia) [inline OpenSim/Actuators/TorqueActuator.h]: Nice catch

Comment (chrisdembia) [inline OpenSim/Actuators/TorqueActuator.h]: Hmm..is there really a "first" and "second" body?

File: OpenSim/Common/Set.h [hunk 2]
```diff
@@ -251,7 +251,7 @@ friend std::ostream& operator<<(std::ostream &aOut,const Set<T> &aSet)
 //=============================================================================
 //_____________________________________________________________________________
 /**
- * Set whether or not this set owns the memory pointed to by the pointers
+ * %Set whether or not this Set owns the memory pointed to by the pointers
```
Comment (chrisdembia) [inline OpenSim/Common/Set.h]: haha nice

**Comment by chrisdembia** [comment]
I made one comment that you can consider addressing; feel free to leave it as is. Merge at will.

File: OpenSim/Actuators/TorqueActuator.h (hunk 1 above)
Comment (tkuchida) [inline OpenSim/Actuators/TorqueActuator.h]: >Hmm..is there really a "first" and "second" body?

From the class description: "A TorqueActuatorr [sic… I'll fix the typo] applies equal and opposite torques on the two bodies (bodyA and B) that it connects."

**Comment by tkuchida** [comment]
>I made one comment that you can consider addressing; feel free to leave it as is. Merge at will.

Thanks. I'll add `(bodyA)` and `(bodyB)` to avoid any confusion.

File: OpenSim/Actuators/TorqueActuator.h (hunk 1 above)
Comment (chrisdembia) [inline OpenSim/Actuators/TorqueActuator.h]: I guess I meant: are the bodies actually ordered in a meaningful way? "first" and "second" might imply that one body "comes first", like maybe it's closer to the root node of the tree.

I don't think it's a big deal. We use a mix of `body*.1` and `body*.A` throughout the code I think.

Comment (tkuchida) [inline OpenSim/Actuators/TorqueActuator.h]: >I guess I meant: are the bodies actually ordered in a meaningful way?

It matters which body is bodyA for two reasons: the sign and the frame in which the torque is expressed. I don't think referring to bodyA as *the first of two bodies to which torque is applied* is confusing.

**Pull request merged**